src/components/ListItem/ListItem.scss

```
@@ -1,4 +1,9 @@
+@import '../../utils/index';
+
 .moonstone-listItem {
+    @include truncate;
+
+    width: 100%;
     padding: 4px 8px;
```

I know it's out of scope, but one day we will have to use token spacing here:
```suggestion
    padding: var(--spacing-nano) var(--spacing-small);
```